pkg/aws/eni/node.go

```diff
@@ -544,6 +556,19 @@ func (n *Node) ResyncInterfacesAndIPs(ctx context.Context, scopedLog *logrus.Ent
 	n.mutex.Lock()
 	n.enis = map[string]eniTypes.ENI{}
 
+	// 1. This calculates the base interface effective limit on this Node, given:
+	// 		* IPAM Prefix Delegation.
+	// 		* Node Spec usePrimaryAddress being enabled.
```

Comment:
Nit for periods on fragments
```suggestion
	// 		* IPAM Prefix Delegation
	// 		* Node Spec usePrimaryAddress being enabled
```